### PR TITLE
Adding new Aluminium alloys

### PR DESCRIPTION
**Pull request by aditeyashukla**
New materials added:

1. Al-2024-T4: Used in 737, 747 and A380
2. Al-Li-1441: Al-Li alloy used in Airbus A380-800 and A380-800F

### CODE REVIEW

**Comment by aditeyashukla** [comment]
Will add more alloys and composite fibers "soon"

File: src/material_data/MaterialProperties.toml [hunk 1]
```diff
@@ -70,6 +69,73 @@
...
+    thermal_conductivity = 121 #W/(m K) # Using Al-2024 since not specified
+
+[Al-2324-T39]
+    description = """Alloy 2324-T39 is a higher strength version of alloy 2024-T351 and is a purity controlled composition variant of alloy 2024. 
+    It was developed for tension-dominated, fatigue and fracture critical plate applications. The T39 temper, developed through special fabricating 
+    practices, improves both strength and fracture toughness properties over alloy 2024 plate. Alloy 2324-T39 plate is being used successfully on 
+    lower wing skin and center wing box components of commercial aircraft """
+    source = "https://www.lookpolymers.com/polymer_Alcoa-2324-T39-Aluminum-Plate.php"
+    density = 2770.0 #kg/m3
+    youngs_modulus = 72.4 e9#Pa
```
Comment (speth) [inline src/material_data/MaterialProperties.toml]: How is this space in `72.4 e9` parsed? There's another one below in the definition of Al-7050-T7 that probably also be removed.

Comment (aditeyashukla) [reply src/material_data/MaterialProperties.toml]: Fixed that in [this](https://github.com/MIT-LAE/TASOPT.jl/pull/75/commits/d96421c2082fa2293d8735b145c01c46702a33b9) commit

Comment (speth) [reply src/material_data/MaterialProperties.toml]: Ah, good. I don't know why GitHub specifically wanted to show me the wrong version of that file.

**Pull request merged**